src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,153 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public static class Node<T> {
```

Comment:
```suggestion
    private static class Node<T> {
```

replace to the end of class